### ISSUE TITLE
[Cache] Define multiple Memcached servers in framework.yaml

### ISSUE THREAD

**Issue by websirnik** [symfony/symfony#27855]
At the moment it's not possible to define multiple Memcached servers in framework.yaml. Would be useful to extend this ability to be able to do either: 
```
framework:
    cache:
        default_memcached_provider: memcached://server1:1111,server2:2222
```
or 
```
framework:
    cache:
        default_memcached_provider:
            - memcached://server1:1111
            - memcached://server2:2222
```

**Comment by websirnik** [comment]
Yes, I'm creating a connection via service at the moment. Two shortcomings of custom service are:
1. I can't create Cache Pool(i might just haven't figured out how)
2. Service requires extra logic to configure different instances with options for different environments where config.yml would simplify that.

**Comment by nicolas-grekas** [comment]
> I can't create Cache Pool(i might just haven't figured out how)

does that work?
```yaml
framework:
    cache:
        default_memcached_provider: your_memcached_connection_service
```

> Service requires extra logic to configure different instances with options for different environments where config.yml would simplify that.

can't you use env vars in `your_memcached_connection_service` do to that?

if those two answers are OK to you, do we still need a config option? (the less options the better usually ;) )

**Comment by nicolas-grekas** [comment]
Related to #28300 and #28175 I suppose, which also ask for multi-server setups for Redis.
A similar DSN format for both backends would be great BTW.

**Comment by nicolas-grekas** [comment]
See proposal in https://github.com/symfony/symfony/pull/28300#issuecomment-420032183

**Comment by nicolas-grekas** [comment]
Implemented in #28598
